games-fps/urbanterror/urbanterror-4.3.3_p20180410.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit eapi7-ver flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="Hollywood tactical shooter based on the ioquake3 engine"
+HOMEPAGE="http://urbanterror.info https://github.com/mickael9/ioq3"
```

Comment:
the first page redirects to https, so please update to reflect it

Reply:
Fixed. I think they added HTTPS lately. thanks for pointing it out!